### DISCUSSION TITLE
半導體史上最嚴峻缺工潮！衝擊為何？人才在哪？一次看懂

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#742] [General]
隨者科技的進步 對於科技人才的需求 不減反增 台灣半導題龍頭台積電也不例外 從上游的IC設計工程師 一路到中 下游的製程及設備工程師都有 也期待台灣未來的科技發展

https://tw.news.yahoo.com/%E5%9C%96%E8%A7%A3-%E8%AA%A0%E5%BE%B5%E7%99%BE%E8%90%AC%E5%B9%B4%E8%96%AA%E5%93%A1%E5%B7%A5-%E5%8D%8A%E5%B0%8E%E9%AB%94%E6%8E%80%E6%90%B6%E4%BA%BA%E5%A4%A7%E4%BD%9C%E6%88%B0-%E6%96%87%E7%B5%84-%E9%AB%98%E8%81%B7%E7%94%9F%E9%83%BD%E8%83%BD%E4%BE%86-022656799.html

**Comment by AnAnistired** [comment]
科技業大好—>人才大缺—>薪水越來越高，電資人生勝利提早退休

**Comment by williamyen0623** [comment]
找一個電資專業的另一半，不想努力了

**Comment by Pietra1226** [comment]
不知道缺工潮會持續到什麼時候，希望到畢業還有QQ

**Comment by ruthchen1020** [comment]
希望台灣可以跟上電動車風潮，打造台灣牌子的電動車，像gogoro這樣的汽車品牌。

**Comment by StevenWanglolz** [comment]
未來人才在這裡，請記得錄取我謝謝
